core-bundle/src/Resources/contao/library/Contao/Automator.php

```diff
@@ -14,6 +14,7 @@
 use FOS\HttpCacheBundle\CacheManager;
 use Symfony\Component\Console\Input\ArgvInput;
 use Symfony\Component\Console\Output\NullOutput;
+use Contao\CoreBundle\Monolog\ContaoContext;
```

Comment:
Where in in this files gets the `ContaoContext` class used?

Reply:
In RC1 the error ```Attempted to load class "ContaoContext" from namespace "Contao".
Did you forget a "use" statement for e.g. "Contao\CoreBundle\Asset\ContaoContext" or "Contao\CoreBundle\Monolog\ContaoContext"?``` will be thrown in line 152.

However, just saw it was fixed 20 day ago in https://github.com/contao/contao/commit/886fba0ff4925dc22698dc7abf9ba1b1eebdf761#diff-55b7dfefbb2703c2fb23f7d50a40ae3ac9a45d88fec605273593116a96e2969eR151